File: Modul 2/Soal 3/PRAK203-2310817210019-NaufalElyzar.c
#include <stdio.h>

int main() {
    float a, b, i, j, x, y;

    scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);

    float hasilnya = (a - b) * i / j - (x + y);

    printf("%.3f", hasilnya);

    return 0;
}

File: Modul 2/Soal 3/PRAK203-2310817210019-NaufalElyzar.py
input_str = input("Masukkan nilai a, b, i, j, x, dan y (dipisahkan oleh spasi): ")
a, b, i, j, x, y = map(float, input_str.split())

hasilnya = a - (b * i / j) - x + y

print(f"Hasilnya adalah {hasilnya:.3f}")
